### PR TITLE
Randomise videos

### PR DESCRIPTION
**Pull request by naomilwx**
Randomise videos that is shown... Can't think of any more weird videos so feel free to add on =P

### CODE REVIEW

File: generate.js [hunk 1]
```diff
@@ -69,7 +69,9 @@ function setCellValue(card, value) {
 }
 
 function win() {
-	$("#video").append($("<iframe width='560' height='315' src='https://www.youtube.com/embed/HRerwXWTRjM?start=35&autoplay=1' frameborder='0' allowfullscreen></iframe>"));
+  var videos = ["https://www.youtube.com/watch?v=dQw4w9WgXcQ?start=35&autoplay=1", "https://www.youtube.com/embed/HRerwXWTRjM?start=35&autoplay=1", "https://www.youtube.com/watch?v=Ct6BUPvE2sM?start=35&autoplay=1"];
```
Comment (sohymg) [inline generate.js]: does your videos need to start at 35 seconds?

Comment (naomilwx) [reply generate.js]: oh wait didn't notice that I copied that over too

Comment (abeltay) [reply generate.js]: Gotta fix that indentation

Comment (abeltay) [reply generate.js]: You might want to embed the videos too

File: generate.js [hunk 2]
```diff
@@ -69,7 +69,9 @@ function setCellValue(card, value) {
 }
 
 function win() {
-	$("#video").append($("<iframe width='560' height='315' src='https://www.youtube.com/embed/HRerwXWTRjM?start=35&autoplay=1' frameborder='0' allowfullscreen></iframe>"));
+	var videos = ["https://www.youtube.com/embed/HRerwXWTRjM?start=35&autoplay=1", "https://www.youtube.com/embed/Ct6BUPvE2sM?autoplay=1"];
```
Comment (fonglh) [inline generate.js]: Let's abstract the video list to a separate file like the word list

**Review by fonglh — CHANGES_REQUESTED** [review]

File: generate.js (hunk 2 above)
Comment (naomilwx) [reply generate.js]: Ok done

**Review by fonglh — APPROVED** [review]

**Pull request merged**